src/sending/SendByEmail.java

```diff
@@ -0,0 +1,17 @@
+package sending;
+
+import interfaceticket.Sender;
+import interfaceticket.Shareable;
+import userANDticket.Ticket;
+
+public class SendByEmail implements Sender {
+    @Override
+    public void send(Shareable shareable) {
+        if (shareable instanceof Ticket) {
```

Comment:
From my perspective no need to have this check here, we're already agreed that all Shareable could be send via SendByEmail. 
Seems like you need extra method inside Shareable that we could call here,  but for now it's fine to leave just System.out.println("Sending via email");